src/main/pages/workspace-admin/what-are-workspaces.adoc

```diff
@@ -10,45 +10,49 @@ folder: workspace-admin
 
 [id="environment"]
 == Environment
 
-Workspace runtime environment is a set of machines where each machine is defined by a recipe. Environment is considered to be healthy when all machines have successfully started and associated installers executed their jobs. Environment is defined by a recipe that can have different types. It is up to environment (and underlying infra) to validate a recipe.
+The workspace runtime environment is a set of machines where each machine is defined by a recipe. The environment is healthy when all the machines successfully start up and the installers execute jobs. The environment is defined by a recipe that can have different types.  The environment and infrastructure validates a recipe. 
 
 [id="machine"]
 == Machine
 
-Runtime environment has minimum one machine that can be (at this moment) a Docker container or a Kubernetes pod. You can create multi machine environments with as many machines as your project infrastructure requires. Though each machine has its own configuration and start policy, it is an integral part of environment, and start failures/crashes of one machine may signal about an entire environment being unhealthy. Machines can freely communicate in a common environment using internal network (e.g. by `service:port`).
+The runtime environment has a minimum of one machine that can be a Docker container or a Kubernetes pod. You can create multi-machine environments with as many machines as your project infrastructure requires. Each machine has a configuration and start policy.  Machine crashes and start failures are signs of an unhealthy environment.  Machines communicate by using the internal network, `service:port`.
 
 [id="recipe"]
 == Recipe
 
-Workspace environment is defined by a recipe that can be a single Docker image, a Dockerfile, a Docker Compose file, or a Kubernetes list of objects with multiple pods and services.
+A workspace environment is defined by a recipe.  The recipe can be one of the following:
+
+* single Docker image
+* Docker file
+* Docker Compose file
+* Kubernetes list of objects with multiple pods and services
 
 [id="bootstrapper"]
 == Bootstrapper
 
-Inside every machine, a bootstrapper is started. Its role is to start installer scripts that will install chosen software/components into a machine. Bootstrapper is a tiny binary compiled from go code. It starts with a set of parameters and a config file with the list of installer scripts to be executed (see below). Launch of a bootstrapper is the first process executed in a machine after `CMD` or `ENTRYPOINT`.
+The bootstrapper starts the installer script after the first process is executed in the machine following the `CMD` or `ENTRYPOINT`.   The role of the bootstrapper is to start the installer scripts with a set of parameters and a configuration file. The bootstrapper is a tiny binary compiled from Go code.
```

Comment:
* the `CMD` or `ENTRYPOINT` -> the `CMD` or `ENTRYPOINT` instructions
* tiny -> small